nomad/grapl-core.nomad

```diff
@@ -0,0 +1,428 @@
+# For rust logging output
```

Comment:
Rather than use comments to describe the variables, it would be good to use the `description` field instead.

Reply:
Yup! My bad :P